gen_bot_db.sh

```diff
@@ -29,6 +29,26 @@ verify_inputs() {
   fi
 }
 
+request_access_token() {
+  client_id=$(pass api.eclipse.org/client_id)
+  client_secret=$(pass api.eclipse.org/client_secret)
+  access_token=$(curl -s --request POST \
+    --url 'https://accounts.eclipse.org/oauth2/token' \
+    --header 'content-type: application/x-www-form-urlencoded' \
+    --data 'grant_type=client_credentials' \
+    --data "client_id=${client_id}" \
+    --data "client_secret=${client_secret}" \
+    --data 'scope= eclipsefdn_view_all_profiles' | jq .access_token | tr -d '"')
+}
+
+query_email_by_user_id() {
+    local user_id="$1"
+    email_address=$(curl -s --request GET \
+      --url "https://api.eclipse.org/account/profile/${user_id}.json" \
```

Comment:
`.json`? Could a `-H "Accept: application/json"` be better?

Reply:
According to Chris, the "." (dot) in the profile name is not accepted. Adding ".json" is a workaround for that. @chrisguindon can you comment on this, if Mikael's suggestion would achieve the same result?